### DISCUSSION TITLE
Interpretation OF THE RESULTS

### DISCUSSION THREAD

**Discussion by treasure316** [korem-lab/SGVFinder2#9] [General]
I'm sorry to bother you.I don't quite understand the results.
Does 1 in the dSV result indicate the presence or absence of this segment?

**Comment by talkorem** [comment]
Hi, 1 is presence, 0 is absence.

**Comment by jiushao12345** [comment]
> I'm sorry to bother you.I don't quite understand the results. Does 1 in the dSV result indicate the presence or absence of this segment?

Hi, @treasure316, I having some problems in SGVFinder2 step. I want ask you for help in creating database step.
